build/css/main.css

```diff
@@ -16142,39 +16142,129 @@ textarea.form-control-lg {
 }
+.govcy-form .govcy-form-control .govcy-checkbox,
+.govcy-form .govcy-form-control .govcy-radio-btn {
+  display: block;
+  position: relative;
+  padding: 6px 16px 8px 60px;
+  margin-top: 12px;
+  cursor: pointer;
+  font-size: 22px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+.govcy-form .govcy-form-control .govcy-checkbox .govcy-checkbox-input, .govcy-form .govcy-form-control .govcy-checkbox .govcy-radio-input,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-checkbox-input,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-radio-input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+.govcy-form .govcy-form-control .govcy-checkbox .govcy-checkbox-input:checked ~ .govcy-tickbox, .govcy-form .govcy-form-control .govcy-checkbox .govcy-checkbox-input:checked ~ .govcy-radio, .govcy-form .govcy-form-control .govcy-checkbox .govcy-radio-input:checked ~ .govcy-tickbox, .govcy-form .govcy-form-control .govcy-checkbox .govcy-radio-input:checked ~ .govcy-radio,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-checkbox-input:checked ~ .govcy-tickbox,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-checkbox-input:checked ~ .govcy-radio,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-radio-input:checked ~ .govcy-tickbox,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-radio-input:checked ~ .govcy-radio {
+  outline: 4px solid #ffdd00;
+  border: 4px solid #000000;
+}
+.govcy-form .govcy-form-control .govcy-checkbox .govcy-checkbox-input:checked ~ .govcy-radio, .govcy-form .govcy-form-control .govcy-checkbox .govcy-radio-input:checked ~ .govcy-radio,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-checkbox-input:checked ~ .govcy-radio,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-radio-input:checked ~ .govcy-radio {
+  outline: none;
+  box-shadow: 0 0 0 4pt #ffdd00;
+}
+.govcy-form .govcy-form-control .govcy-checkbox .govcy-checkbox-input:checked ~ .govcy-tickbox:after, .govcy-form .govcy-form-control .govcy-checkbox .govcy-checkbox-input:checked ~ .govcy-radio:after, .govcy-form .govcy-form-control .govcy-checkbox .govcy-radio-input:checked ~ .govcy-tickbox:after, .govcy-form .govcy-form-control .govcy-checkbox .govcy-radio-input:checked ~ .govcy-radio:after,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-checkbox-input:checked ~ .govcy-tickbox:after,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-checkbox-input:checked ~ .govcy-radio:after,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-radio-input:checked ~ .govcy-tickbox:after,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-radio-input:checked ~ .govcy-radio:after {
+  display: block;
+}
+.govcy-form .govcy-form-control .govcy-checkbox .govcy-tickbox,
+.govcy-form .govcy-form-control .govcy-radio-btn .govcy-tickbox {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 44px;
```

Comment:
To be consistent the `text` (but not the text area) input must also have height:44px;